apps/sharedtest/test/unit/event_safety_test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+/* global eventSafety */
+
+require('/shared/js/event_safety.js');
+
+suite('eventSafety', function() {
+
+  test('fires after event', function(done) {
+    this.sinon.useFakeTimers();
+    eventSafety(window, 'foobar', done, 1000);
+    window.dispatchEvent(new CustomEvent('foobar'));
+
+    // Tick the clock to ensure that the timeout is cleared.
+    // Sinon will throw an error if done is called multiple times.
+    this.sinon.clock.tick(1000);
```

Comment:
can we move those into a teardown, I feel like it makes the test much more expressive